docs/plan-for-production.md

```diff
@@ -272,13 +277,14 @@ However, for best performance and fault isolation, you should deploy individual
 KV store instances for container networks and Swarm discovery. This is
 especially so in demanding business critical production environments.
 
-Engine container networks also require version 3.16 or higher of the Linux
-kernel. Higher kernel versions are usually preferred, but carry an increased
-risk of instability because of the newness of the kernel. Where possible, you
-should use a kernel version that is already approved for use in your production
-environment. If you do not have a 3.16 or higher Linux kernel version approved
-for production, you should begin the process of getting one as early as
-possible.
+Container networks in docker 1.9.x also require version 3.16 or higher of the 

```

Comment:
```markdown
Beginning with Docker Engine 1.9, Docker container networks require specific
Linux kernel versions. Higher kernel versions are usually preferred, but carry
an increased risk of instability because of the newness of the kernel. Where
possible, use a kernel version that is already approved for use in your
production environment. If you can not use a 3.10 or higher Linux kernel version
for production, you should begin the process of approving a newer kernel as
early as possible.
```